lib/sqs.test.js

```diff
@@ -427,10 +434,824 @@ describe('sqs', () => {
       createQueue(params2);
       const listOfQueues = listQueues();
 
-      expect(listOfQueues).toEqual([
-        'https://queue.amazonaws.com/123/foo-bar1',
-        'https://queue.amazonaws.com/123/foo-bar2'
+      expect(listOfQueues).toEqual(['foo-bar1', 'foo-bar2']);
+    });
+  });
+
+  describe('sendMessage', () => {
+    test('throw error when queue does not exist', () => {
+      let error;
+      try {
+        sendMessage({ QueueUrl: 'http://localhost:3000/queues/foo' });
+      } catch (err) {
+        error = err;
+      }
+
+      expect(error.message).toEqual('AWS.SimpleQueueService.NonExistentQueue; see the SQS docs.');
+      expect(error.code).toEqual('AWS.SimpleQueueService.NonExistentQueue');
+    });
+
+    test('successfully send message when queue has no messages', () => {
+      createQueue({ QueueName: 'foo-bar' });
+      const params = {
+        MessageBody: 'some message',
+        QueueUrl: 'http://localhost:3000/queues/foo-bar'
+      };
+      const response = sendMessage(params);
+      expect(response).toEqual({
+        MD5OfMessageBody: 'df49b60423903e095b80d9b4a92eb065',
+        MessageId: expect.any(String)
+      });
+    });
+
+    test('successfully send message when queue has messages', () => {
+      createQueue({ QueueName: 'foo-bar' });
+      const params1 = {
+        MessageBody: 'first message',
+        QueueUrl: 'http://localhost:3000/queues/foo-bar'
+      };
+      const params2 = {
+        MessageBody: 'second message',
+        QueueUrl: 'http://localhost:3000/queues/foo-bar'
+      };
+
+      sendMessage(params1);
+      const response = sendMessage(params2);
+      expect(response).toEqual({
+        MD5OfMessageBody: 'bf48a9b23ca5015653edebff31d6b879',
```

Comment:
I think it would demonstrate a safer test if we mocked md5() to return a specific string and assert on that specific string.

Otherwise, this assertion is as good as as `.toBe(expect.any(String))`